static/app/components/actions/resolve.tsx

```diff
@@ -215,7 +249,13 @@ function ResolveActions({
             ? ['next-release', 'current-release', 'another-release']
             : []
         }
-        menuTitle={t('Resolved In')}
+        menuTitle={
```

Comment:
Seems a bit hacky but idk if there's a better way

Reply:
yeah not my favorite but it does work really well

Reply:
items is much more difficult to use

Reply:
I guess the alternative is not using the dropdown menu at all. You could use DropdownTrigger and `useOverlay` for this case. Probably not be worth the effort if this works well